src/handlers/census/events/DeathEvent.ts

```diff
@@ -116,10 +115,6 @@ export default class DeathEvent {
 
         this.isHeadshot = event.is_headshot;
 
-        // Additional logic to figure out things Census doesn't tell us...
-        this.isSuicide = this.characterId === this.attackerCharacterId;
-
-        // TK logic, does NOT include NSO. If same faction but not same character
-        this.isTeamkill = this.attackerFaction === this.characterFaction && !this.isSuicide;
+        this.killType = event.kill_type;
```

Comment:
Have you rigourlessly tested this? Wouldn't want a repeat of what we had before where every kill was a headshot :-/

Reply:
This is a computed property not a casted. So basically a function which will return a result but as a getter.